Kconfig.zephyr

```
@@ -344,6 +344,47 @@ config BUILD_OUTPUT_S19
 	  Build an S19 binary zephyr/zephyr.s19 in the build directory.
 	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
 
+config BUILD_OUTPUT_UF2
+	bool "Build a binary in UF2 format"
+	depends on BUILD_OUTPUT_BIN
+	help
+	  Build a UF2 binary zephyr/zephyr.uf2 in the build directory.
+	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
+
+if BUILD_OUTPUT_UF2
+
+config BUILD_OUTPUT_UF2_FAMILY_ID
+	# Prompt less config to setup a list of default UF2 Family IDs
```

It should no longer be prompt less, as I suppose users should still be able to provide a custom string if current soc is not known. So you probably want to re-add.

```suggestion
	string "UF2 Device Family" 
```
Note: The prompt less was only in case we wanted to have the defaults located outside of `Kconfig.zephyr`

---

Ok. Prompt restored.